### DISCUSSION TITLE
Setup taiko and gauge test in Docker

### DISCUSSION THREAD

**Discussion by ramesh-khatri** [getgauge/taiko#1791] [General]
I am getting this error while running my built container
<img width="586" alt="docker_error" src="https://user-images.githubusercontent.com/76654941/104566858-3cbbc380-5676-11eb-9846-24a93265994b.png">
Can anyone give me any suggestion or feedback on what mistake am I doing?

**Comment by sriv** [comment]
Check your `logs` directory for possible errors. If you can share a sample, we can investigate it better.

**Reply by ramesh-khatri** [reply]
Here is the log file.
[2021-01-14T09_48_07_838Z-debug.log](https://github.com/getgauge/taiko/files/5829024/2021-01-14T09_48_07_838Z-debug.log)

**Reply by sriv** [reply]
That is not the log I was referring to. You'll find gauge logs in the `logs` directory under the project root directory

**Reply by sriv** [reply]
ok, typically `Runner is not Alive` means either the runner could not be started or the runner process crashed for some reason. Check your project properties/environment for `logs_directory`. This should have the location where the logs are written.

Also, are you able to run the project outside docker?

**Reply by ramesh-khatri** [reply]
@sriv , Thanks, I read the log file and found that the issue was due to path separator. I fix it and now it is running well.
